fury/actors/tensor.py

```diff
@@ -0,0 +1,213 @@
+import os
+
+import numpy as np
+
+from fury import actor
+from fury.shaders import (attribute_to_actor, import_fury_shader,
+                          shader_to_actor, compose_shader)
+
+
+def tensor_ellipsoid(centers, axes, lengths, colors, scales, opacity):
+    """
+    Visualize one or many Tensor Ellipsoids with different features.
+
+    Parameters
+    ----------
+    centers : ndarray(N, 3)
+        Tensor ellipsoid positions
+    axes : ndarray (3, 3) or (N, 3, 3)
+        Axes of the tensor ellipsoid
+    lengths : ndarray (3, ) or (N, 3)
+        Axes lengths
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,), optional
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    scales : float or ndarray (N, ), optional
+        Tensor ellipsoid size, default(1)
+    opacity : float, optional
```

Comment:
We might  need to create an issue for that. `Opacity` and `RGBA` are confusing. who has the priority? it seems that opacity have the priority.  We might need to remove `opacity` parameter.

Reply:
I agree. For now, I will avoid the alpha channel, leave the color as ndarray (N,3) or tuple (3,) and keep the opacity.